app/models/manageiq/providers/hawkular/inventory/parser/middleware_manager.rb

```diff
@@ -44,10 +44,7 @@ def fetch_middleware_servers
                 set_lives_on(server, container) if container
               end
             else
-              machine_id = collector.machine_id(eap.feed)
-              host_instance = find_host_by_bios_uuid(machine_id) ||
-                              find_host_by_bios_uuid(alternate_machine_id(machine_id))
-              set_lives_on(server, host_instance) if host_instance
+              associate_with_rhevm(server, eap.feed)
```

Comment:
is this linking only to VMs in rhevm? afaik the functionality of linkage existing prior to this patch works for vmware VMs as well.